app/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.5
+
+ADD . /app
```

Comment:
I'm not a Docker expert, but here's the output from an online Dockerfile Linter, <http://hadolint.lukasmartinelli.ch>:

> **DL4000** Specify a maintainer of the Dockerfile
> **DL3020** Use COPY instead of ADD for files and folders

There is also http://dockerfile-linter.com/?lang=en which gives a bit more output, notably:

> INFO:  at 5:  Adjacent RUN can be merged with previous RUN to reduce image layer number.